tools/tox.ini

```diff
@@ -9,31 +6,17 @@ skipsdist = true
 
 [testenv]
 passenv =
+    CUDA_PATH
+    CUDAARCHS
+    NUM_PROCESSES
     PIP_USER
     PYTHONUSERBASE
+    TEST_DATA_PATH
```

Comment:
I saw that you removed the `datatest` marker. Do we still need to pass this variable?

Reply:
Maybe it is good to keep it anyway, in case datatest are added in the future.

Reply:
Yes, after speaking to @halungge I realised that the `datatest` marker is superfluous here. The serialised data is accessed through a fixture not the marker, so in short yes we still need to pass the variable.